store.go

```diff
@@ -8,8 +8,8 @@ import (
 	_ "github.com/go-sql-driver/mysql"
 	_ "github.com/lib/pq"
 
-	"github.com/mattermost/mattermost-server/v6/plugin"
-	"github.com/mattermost/mattermost-server/v6/shared/driver"
+	"github.com/mattermost/mattermost/server/public/plugin"
+	"github.com/mattermost/mattermost/server/v8/platform/shared/driver"
```

Comment:
With the removal of Playbooks and Board from the mono repo, `github.com/mattermost/mattermost/server/v8/platform/shared/drive` is not longer imported by any packages in the mono repo. Any objections to moving it into `public`?